### ISSUE TITLE
Rendering of square root in weasy print

### ISSUE THREAD

**Issue by julianstirling** [mbarkhau/markdown-katex#5]
Hi,

I am having problems with square roots, even when I try to follow the instructions in the README. I am just running the test formula

```math
f = \sqrt{\frac{2}{3}}
```

In the browser the square root only appears if I don't set `'no_inline_svg': True,`. In WeasyPrint the square root does not appear for  `no_inline_svg` being True or False.

## Details
**OS**  
Kubuntu 20.04

**Python**  
Python 3.7.10

**Packages**
```
WeasyPrint==52.5
Markdown==3.3.2
markdown-katex==202105.1031
```

## Edited to add

It seems that the issue is my CSS. Specifically `object-fit` being set to contain. It would be good to not have to add classes to every other image as this ruins the simplicity of markdown. The fix I can do in the browser is to  change the katex css to

```css
.katex img {
    object-fit: fill;
    padding: unset;
}
````

I will open an MR adding these to the inline CSS.

**Comment by julianstirling** [comment]
It can be fixed in my css too...

**Comment by mbarkhau** [comment]
> Is it possible to do this in `markdown-katex`?

The option ` 'insert_fonts_css': True` [injects inline css into the html output](https://github.com/mbarkhau/markdown-katex/blob/d4b8b226d728b4c8dd1e0883e79de6c0214d97b3/src/markdown_katex/extension.py#L288) (I guess the naming is not ideal).
Is the idea here to use different/custom css?

**Comment by mbarkhau** [comment]
Fixed by #7
